scripts/experiments_images/DeepSVDD.jl

```diff
@@ -87,7 +87,7 @@ function fit(data, parameters)
 			ae_iters=10000
 		)
 	)
-	svdd = GenerativeAD.Models.conv_ae_constructor(;all_parameters...) |> gpu
```

Comment:
Why was an autoencoder constructor used in the first place?

Reply:
SVDD is, in first stage of training, only AE with additional parameters. So it was mainly naming issue at that time